portability-worker/src/main/java/org/dataportabilityproject/worker/WorkerMain.java

```diff
@@ -61,11 +117,43 @@ public static void main(String[] args) throws Exception {
         System.exit(0);
     }
 
+    // TODO: consider getting this dynamically rather than hardcoding
+    private static List<TransferServiceProvider> listAllTransferServiceProviders() {
+        // TODO if we get these dynamically, consider validating all service names unique
+        return ImmutableList.of(new MicrosoftTransferServiceProvider());
```

Comment:
Since this is a hard coded list for now, does it make sense to make it a map? "serviceProviderName" -> ServiceProvider - that would help with determining uniqueness of service names and make looking up the import and export service providers easier. 

Then later on, we can initialize the map by using the service loader.